client/lib/sites-list/list.js

```diff
@@ -528,9 +573,9 @@ SitesList.prototype.getVisible = function() {
  *
  * @api public
  **/
-SitesList.prototype.getVisibleAndNotRecent = function() {
+SitesList.prototype.getVisibleAndNotRecentNorStarred = function() {
```

Comment:
Our best function name! Agreed, we have so many filtered views now that it makes sense to consolidate. I didn't put much thought into it since we want to move all of this into proper selectors, but could be a good update.